coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 51.41
+    "covered_percent": 55.81
```

Comment:
Just revert that change for now and you should be good. Are you using the same Ruby version as TravisCI does?